ui/apps/platform/src/utils/permissionRuleGroupUtils.js

```diff
@@ -29,7 +29,7 @@ export const getGroupsWithDefault = (groups, authProviderId, roleName, defaultGr
                 authProviderId: defaultGroup.props.authProviderId,
                 id: defaultGroup.props.id,
             },
-            roleName: defaultGroup.roleName,
```

Comment:
what does this change do?

Reply:
The results of `getGroupsWithDefault` will be used as the `newGroups` within `updateOrAddGroup` (the call will be [here within groupSaga.js](https://github.com/stackrox/stackrox/blob/2970103375a89dfe6ce4afcc0f033c68a49abff5/ui/apps/platform/src/sagas/groupSagas.js#L43-L46)).
The `roleName` will be the potentially updated role name, i.e. when you change the minimum access role in the dropdown within the auth provider configuration page. The `defaultGroup.roleName` will be the currently set role name of the default group returned via API. This _may_ be stale. 
With this change, the new, potentially updated role name will be used when doing the `/v1/groupsbatch` request.